HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc

```diff
@@ -301,7 +301,8 @@ TH1F  HLTBTagHarvestingAnalyzer::calculateEfficiency1D(DQMStore::IBooker& ibooke
 		double e;
 		if(d!=0){
 			e=n/d;
-			err =  sqrt(e*(1-e)/d); //from binomial standard deviation
+			err = std::max(e - TEfficiency::ClopperPearson(d, n, 0.683, false), TEfficiency::ClopperPearson(d, n, 0.683, true) - e);
```

Comment:
naive question
why don't you make use of the DQMGenericClient for creating the efficiency plots ?

Reply:
Are you commenting this particular change or the whole calculateEfficiency1D function?
Could you please point me to examples of creating efficiency histograms with the DQMGenericClient?